ietf/doc/views_search.py

```diff
@@ -345,44 +345,35 @@ def ad_dashboard_group(doc):
         if doc.get_state_slug('draft') == 'rfc':
             return 'RFC'
         elif doc.get_state_slug('draft') == 'active' and doc.get_state_slug('draft-iesg'):
-            return '%s Internet-Draft' % doc.get_state('draft-iesg').name
+            return str(doc.get_state('draft-iesg').name)
         else:
-            return '%s Internet-Draft' % doc.get_state('draft').name
+            return str(doc.get_state('draft').name)
     elif doc.type.slug=='conflrev':
         if doc.get_state_slug('conflrev') in ('appr-reqnopub-sent','appr-noprob-sent'):
-            return 'Approved Conflict Review'
+            return 'Approved'
         elif doc.get_state_slug('conflrev') in ('appr-reqnopub-pend','appr-noprob-pend','appr-reqnopub-pr','appr-noprob-pr'):
-            return "%s Conflict Review" % State.objects.get(type__slug='draft-iesg',slug='approved')
+            return str(State.objects.get(type__slug='draft-iesg',slug='approved'))
         else:
-          return '%s Conflict Review' % doc.get_state('conflrev')
+          return str(doc.get_state('conflrev'))
```

Comment:
here, why aren't you just getting the state name?
See https://github.com/ietf-tools/datatracker/blob/601ab53db9619c68b191b8d42902a1e59cb636bd/ietf/doc/models.py#L89C1-L90C25